cluster-scope/base/rbac.authorization.k8s.io/clusterroles/vertical-pod-autoscaler/clusterrole.yaml

```diff
@@ -0,0 +1,23 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: grafana-operator
```

Comment:
should this role be in the `grafana-operator` directory instead of the vpa directory?

Reply:
Actually grafana-operator is supposed to be namespace scoped so you might not need this ClusterRole